packages/bitcore-node/src/modules/ethereum/api/erc20Transform.ts

```diff
@@ -24,6 +24,37 @@ export class Erc20RelatedFilterTransform extends Transform {
       tx.abiType.params[8].value.toLowerCase() === this.tokenAddress.toLowerCase()
     ) {
       tx.value = tx.abiType!.params[0].value as any;
+    } else if (tx.internal && tx.internal.length > 0) {
+      try {
+        const tokenRelatedIncomingInternalTxs = tx.internal.filter(
+          (internalTx: any) => this.tokenAddress.toLowerCase() === internalTx.action.to.toLowerCase()
```

Comment:
There are some transactions with `internal.action.to` == undefined. Need to add a check for the to field